packages/ckeditor5-image/theme/imagestyle.css

```diff
@@ -48,6 +55,13 @@
 		margin-right: auto;
 	}
 
+	/* Simulates margin collapsing with the preceding paragraph, which does not work for the floating elements. */
+	& p + .image-style-align-left,
+	& p + .image-style-align-right
+	& p + .image-style-side, {
```

Comment:
This won't transpile to correct CSS.

Reply:
Why does it not apply to `image-style-block-*` BTW? I'm trying to understand the use case.

Reply:
It took me a while to catch this comma.

It doesn't have to be applied to the block images because they don't have the `float` property, so their margins collapse properly with the preceding paragraph's margins.
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing